src/collection-view.js

```diff
@@ -6,6 +6,7 @@
 // A view that iterates over a Backbone.Collection
 // and renders an individual child view for each model.
 Marionette.CollectionView = Marionette.View.extend({
+  cidPrefix: 'mncv',
```

Comment:
I don't think we should overide the view's cid. It's nice to have a format for that